refinery/ui/source/js/commons/data-sets/search-api.spec.js

```diff
@@ -68,7 +68,7 @@ describe('DataSet.search-api: unit tests', function () {
       'hl.simple.post': '%3C%2Fem%3E',
       'hl.simple.pre': '%3Cem%3E',
       q: _query,
-      qf: 'title%5E0.5+accession+submitter+text',
+      qf: 'title%5E0.5+accession+submitter+text+description',
```

Comment:
For additional context, the search index increased from `1.23Mb` -> `1.31Mb` after indexing with the DataSet description as an `EdgeNGram` field.